add_ia_covers/README.md

```diff
@@ -0,0 +1 @@
+This is still a work in progress
```

Comment:
Going to use the https://github.com/internetarchive/openlibrary-bots/blob/master/README.md template for this (won't fill out everything, just a start):
openlibrary-bots: add_ia_covers
The Open Library believes that "borrowable works should always have bookcover" and to fall back to the IA for this (as explained in the Github #2550 issue). This would include providing covers where aren't any, filling in blank covers, and also changing covers that are just a solid color to one that has a readable title and author on it.

Current Bots:
cover_updater.py - adds a new cover in for ones that do not have one but is on the IA

Documentation file for all the finished editions:
fixed_books.csv